Source/WebKit/UIProcess/Automation/Automation.json

```diff
@@ -937,6 +945,14 @@
                 { "name": "type", "type": "string", "description": "The log entry type." },
                 { "name": "method", "type": "string", "optional": true, "description": "The log entry console method."}
             ]
+        },
+        {
+            "name": "bidiMessageSent",
```

Comment:
The names `processBidiMessage` and `bidiMessageSent` names don't really feel like the call and response that they are. Could we consider `bidiCommand` and `bidiResponse`, which frames both of these from the client's point of view.

Reply:
What about `processBiDiCommand` instead of `processBiDiMessage`, as the methods usually start with a verb and all messages that come from the client are commands?

One drawback of `bidiResponse` is in the case of non-response messages, like events.